src/layouts/PageLayout.jsx

```diff
@@ -18,10 +19,10 @@ const useStyles = makeStyles(them => ({
 const query = graphql`
   query {
     allBuilderModels {
-      header(limit: 1, options: { cachebust: true }) {
+      header(limit: 1) {
```

Comment:
cachebust is needed here (only on the gatsby graphql query to ensure we get the freshest content when on build time)